app/assets/javascripts/admin/all.js

```diff
@@ -26,6 +25,23 @@
 //= require angular-rails-templates
 //= require lodash.underscore.js
 
+// datetimepicker  (fil, nb)
+//= require flatpickr
+//= require flatpickr/l10n/ar
+//= require flatpickr/l10n/cat
+//= require flatpickr/l10n/cy
+//= require flatpickr/l10n/de
+//= require flatpickr/l10n/es
+//= require flatpickr/l10n/fr
+//= require flatpickr/l10n/it
+//= require flatpickr/l10n/nl
+//= require flatpickr/l10n/pl
+//= require flatpickr/l10n/pt
+//= require flatpickr/l10n/ru
+//= require flatpickr/l10n/sv
+//= require flatpickr/l10n/tr
+//= require shortcut-buttons-flatpickr.min
```

Comment:
managing this dependency without a package manager scares me a bit and it's something we've been trying to overcome. 

Rails' historical mechanism it's been to provide gems ready to be used by the asset pipeline such as https://github.com/adambutler/vuejs-rails or https://github.com/rails/jquery-rails. They keep up to date with the JS dependency so we only need to deal with updating the gem.

Including the JS file as-is, this dependency won't be updated until it's too late. That's why we introduced yarn some time ago plugged it into Rails' asset pipeline (as Rails does with its webpack support in future versions). You can take https://github.com/openfoodfoundation/openfoodnetwork/pull/6049 as an example. The goal in the long-term is to have 0 vendorized JS dependencies.

Perhaps @luisramos0 can shed some more light into this.

Reply:
I totally agree and I missed it. Will make the correction! Thanks for pointing that out to me!

Reply:
Commit 6d07fc4 should resolve. Let me know.